src/contracts/implementations/arbitrable/MultipleArbitrable.js

```diff
@@ -90,10 +71,10 @@ class MultipleArbitrable extends ContractImplementation {
       0,
       'latest',
       {
-        _disputeID: disputeId,
         _arbitrator: arbitratorAddress,
+        _disputeID: disputeId,
         _party: this._Web3Wrapper.getAccount(0),
-        _transactionId: this.arbitrableTransactionId
+        _evidence: evidenceJsonLink
```

Comment:
I don't think this is going to work. `_party` and `_evidence` are not indexed. Also the only reason that we are looking up the event is to fetch the `evidenceJsonLink`. You shouldn't be able to pass it to `getEvidence` or else it isn't coming from the blockchain